infrastructure/logging/src/main/java/tech/pegasys/teku/infrastructure/logging/EventLogger.java

```diff
@@ -88,6 +88,10 @@ public void syncCompleted() {
     info("Syncing completed", Color.GREEN);
   }
 
+  public void syncStart() {
+    info("Syncing started", Color.GREEN);
```

Comment:
I'd probably make this one `Color.YELLOW` rather than green - it's indicating that the node is not ready.